### PR TITLE
chore(a11y): re-enable a11y tests

### PR DESCRIPTION
**Pull request by annawen1**
### Related Ticket(s)

N/A

### Description

Pass in `openssl` flag to the `test:a11y` commands and remove the true setting

### Changelog

**New**

- {{new thing}}

**Changed**

- {{changed thing}}

**Removed**

- {{removed thing}}

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,7 +29,7 @@
     "lint:styles": "stylelint '**/*.{css,scss}' --config ./packages/stylelint-config-ibmdotcom",
     "reset": "yarn cache clean && yarn clean && yarn install && yarn build",
     "test": "yarn test:unit && yarn test:a11y && yarn test:integration:build",
-    "test:a11y": "yarn lerna run --stream --prefix test:a11y",
+    "test:a11y": "NODE_OPTIONS=--openssl-legacy-provider yarn lerna run --stream --prefix test:a11y",
```
Comment (ariellalgilmore) [inline package.json:32]: do we need the node_options added it here? the main file should just be calling the react/web-components commands i thought

Comment (annawen1) [reply package.json:32]: cool I'll remove it

**Review by ariellalgilmore — APPROVED** [review]
LGTM!

**Review by kennylam — APPROVED** [review]
LGTM!

**Review by RichKummer — APPROVED** [review]

**Pull request merged**